### ISSUE TITLE
Incomplete uniform loss function

### ISSUE THREAD

**Issue by Tandon-A** [UncleMEDM/PUGAN-pytorch#3]
Hello,

Thank you for this PyTorch implementation of PUGAN. 

In this work for the [uniform loss](https://github.com/UncleMEDM/PUGAN-pytorch/blob/master/loss/loss.py#L30), you have only implemented the U_clutter loss as defined in the paper and not the U_imbalance part. Are there any reasons for using only the clutter part? 

Regards 
Abhishek

**Comment by UncleMEDM** [comment]
It is just a simpler implementation, because when the U_clutter is small, U_imbalance is usually small, and there is no strong motivation to multiply these two terms. And During the experiment, I found out that this simple implementation also perform very well. 

best regards,
Liu Haolin

**Comment by Tandon-A** [comment]
Thank you for providing details.
